cloud/docker_client.go

```diff
@@ -12,23 +12,25 @@ import (
 	"github.com/docker/docker/api/types/network"
 	docker "github.com/docker/docker/client"
 	"github.com/docker/go-connections/nat"
-	"github.com/evergreen-ci/evergreen/model/distro"
 	"github.com/evergreen-ci/evergreen/model/host"
 	"github.com/evergreen-ci/evergreen/util"
-	"github.com/mitchellh/mapstructure"
 	"github.com/mongodb/grip"
 	"github.com/mongodb/grip/message"
 	"github.com/pkg/errors"
 )
 
+// Temporary port used for all connections to the Docker daemon. To be moved to
+// ContainerPoolsConfig section of Evergreen settings after EVG-3598
+const ClientPort = 3389
```

Comment:
Made this a constant for now, since EVG-3598 is moving this to the Container Pool Settings and this will have to be changed.